1.4.0 release

KingLemming opened the pull request.

skyboy commented:
Does this have any performance impact similar to noise1 being implemented in a software renderer on some mac cards?

miuirussia commented:
No, it works flawlessly. I tested on my iMac

KingLemming committed:
Add Gourmand and Disenchantment dynamos.

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Work on Gourmand Dynamo recipes.

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Machine Power config.

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Add slag/rich slag deco variants. Thanks Reda!

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Work on Fluid Reservoir. :)

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Work on simplifying base Thermal tile.

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Tile reorganization.

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Some work on Reservoirs. Not quite there.

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Block and Item asset restructuring.

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Container refactorizin'

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Add SEO tooltips.
Data-drive the Hive Hopper.

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Restructure lang files.

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Texture reorganization.

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Update accesstransformer.cfg
Add Syrup bottle.
Change some storage recipes.

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Add rubber trees - pass 1.

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
RUBBER TREES.

(And underlying energy system configurability.)

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Additional vanilla support for rubber trees.

Signed-off-by: King Lemming <kinglemming@gmail.com>

karakufire committed:
update ja_jp.json

KingLemming committed:
Minor refactors.

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Move Satchels to Core.

Signed-off-by: King Lemming <kinglemming@gmail.com>

karakufire committed:
update ja_jp.json

KingLemming committed:
Add securable recipe.

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Colorable Satchels!
Adjust Rubber Tree texture.

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
AreaEffectTile work.

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
ITileXpHandler refactor.

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Refactor Core module workspace.

Add skeleton for Item Cells.

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Merge pull request #1 from karakufire/ja_jp

update ja_jp.json

LunarFactory committed:
Add ko_kr translations

- Added ko_kr translations by Taki_B and PixelVoxel with Team.PixelQuest. Thank you lemming!

KingLemming committed:
Add Slot Seal item.
Skeleton for Chunk Loader and Item Cell.

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Merge pull request #2 from PixVoxel/main

Add ko_kr translations

KingLemming committed:
Initial Curios work.

Signed-off-by: King Lemming <kinglemming@gmail.com>

covers1624 committed:
Mavenize!

- Move to official mappings.
- Prepare for Maven.
- Refactor generated resources.

KingLemming committed:
Curios and ChunkLoader work.

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Some Item Cell work. WIP.

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
More Item Cell work.
Move Rubber tree to Horizons.

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
AccessTransformer update.

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Add button support to wood registration.

KingLemming committed:
Update README
Fix a recipe

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Elastica -> Rubberwood

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Add CRT for new Dynamos
Improve AreaEffectTiles

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Fluid ingredient work.

Signed-off-by: King Lemming <kinglemming@gmail.com>

jaredlll08 committed:
update removeRecipe to use IIngredient

jaredlll08 committed:
Update CraftTweaker

jaredlll08 committed:
remove unused parameter

jaredlll08 committed:
Revert a premature code change

This wasn't meant to be included in this PR, but a future one

jaredlll08 committed:
Move to CTFluidIngredient

jaredlll08 committed:
Register missing recipe types

jaredlll08 committed:
Forgot to push this change

jaredlll08 committed:
Add missing CRT managers for devices

jaredlll08 committed:
Fix hive extractor output not being copied.

jaredlll08 committed:
Add recipe printing to dynamos

jaredlll08 committed:
Add recipe printing to machines

jaredlll08 committed:
Add recipe printing to devices

jaredlll08 committed:
Add Recipe Replacement support to all managers that can support it

jaredlll08 committed:
Make the unsupported recipe handlers stop erroring

jaredlll08 committed:
Add device examples

jaredlll08 committed:
Add dynamo examples

jaredlll08 committed:
Add machine examples

KingLemming committed:
Merge pull request #3 from jaredlll08/feature/modernizeCRT

Modernize CraftTweaker integration

KingLemming committed:
Merge pull request #4 from jaredlll08/feature/CTFluidIngredient

Make CraftTweaker support use CTIngredient

KingLemming committed:
Merge pull request #5 from jaredlll08/fix/registerRecipeTypes

Register missing recipe types

KingLemming committed:
Merge pull request #6 from jaredlll08/feature/addDeviceCRTManagers

add device crt managers

KingLemming committed:
Merge pull request #7 from jaredlll08/fix/non-cloning-hive-extractor

Fix hive extractor output not being copied.

KingLemming committed:
Merge pull request #8 from jaredlll08/features/RecipePrintingReplacing

Add Recipe Printing and Replacing support

KingLemming committed:
Merge pull request #9 from jaredlll08/features/crafttweaker-examples

Add Crafttweaker example scripts

KingLemming committed:
Refactors.

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Refactors/cleanup.

KingLemming committed:
Adjust internal field names.

Signed-off-by: King Lemming <kinglemming@gmail.com>

jaredlll08 committed:
add support for IIngredientWithAmount

KingLemming committed:
Merge pull request #10 from jaredlll08/feature/CRTIIngredientWithAmount

add support for IIngredientWithAmount

KingLemming committed:
Code cleanup.

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Fix mcmeta file.
Adjust some textures.
Item Cell still in progress.

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Remove compat recipes.
Adjust machine I/O logic.

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Adjust Satchel for locked slots.

Signed-off-by: King Lemming <kinglemming@gmail.com>

EvanHsieh0415 committed:
Upload zh_TW.lang & Fix one Translation error

EvanHsieh0415 committed:
Edit Nickel "黃鐵" -> "鎳"

KingLemming committed:
Change Augmentable tile behavior w/ right clicks.

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Adjust recipe tooltips.

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Improve JEI descriptiveness.

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Improve right-click augment installation.

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Fix Machine configs, remove extraneous method from TinkerBenchTile

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Consistency refactors w/ BlockItems.

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Adjust Cell behavior for consistency.

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Add tool configuration options!

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Add Netherite items and recipes. Remove item cell.

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Beta release staging.

Signed-off-by: King Lemming <kinglemming@gmail.com>

karakufire committed:
update ja_jp.json

karakufire committed:
update ja_jp.json

KingLemming committed:
Add Foods/Crops ItemGroup.

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Move Blocks to proper Creative Tabs.

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Fluid Cell render changes.

Satchel blacklist work.

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Guidebook refactors and new entry work.
Satchel denylist!

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
More Patchouli entries and edits.
Refactors to support RF energy independence!

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Merge pull request #11 from EvanHsieh0415/main

Fix One Translation Error

KingLemming committed:
Merge pull request #12 from karakufire/ja_jp

update ja_jp.json
